packages/relic/test/ssl_certs.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
This file will need to be excluded in `.pubignore` or relic may not publish to pub (see the auth package).